crypto/x509/v3_tlsf.c

```diff
@@ -135,5 +135,6 @@ static TLS_FEATURE *v2i_TLS_FEATURE(const X509V3_EXT_METHOD *method,
 
  err:
     sk_ASN1_INTEGER_pop_free(tlsf, ASN1_INTEGER_free);
+    ASN1_INTEGER_free(ai);
```

Comment:
I think this could lead to a double free if we go around the "for" loop multiple times, and then get an error before the line where we reassign `ai` to a new integer. Probably we need to additionally set `ai = NULL;` after the successful push.

Reply:
Good point.

Reply:
Done